msbuild/tests/Xamarin.iOS.Tasks.Tests/ProjectsTests/ReleaseBuild.cs

```diff
@@ -20,7 +20,7 @@ public void BuildTest ()
 			BuildProject ("MyReleaseBuild", Platform, "Release");
 		}
 
-		[Ignore] // requires msbuild instead of xbuild
+		[Ignore ("requires msbuild instead of xbuild")]
```

Comment:
maybe that should not be ignored anymore ?

Reply:
Yes, but that's for another PR.

Reply:
Agreed. Maybe file an issue so we don't forget?

Reply:
https://github.com/xamarin/xamarin-macios/issues/7609